### PR TITLE
Various changes 8.3.2023

### PR DESCRIPTION
**Pull request by flichtenheld**
- t_client_ips.rc fixes for buildbot
- vcpkg S3 cache for msibuild
- switch jenkins-agent to use SSH method instead of agent method

### CODE REVIEW

File: jenkins/windows-server/shared.pkr.hcl [hunk 1]
```diff
@@ -43,6 +44,7 @@ source "amazon-ebs" "windows-server-2019" {
   communicator     = "winrm"
   force_deregister = true
   instance_type    = "c5a.xlarge"
+  iam_instance_profile = var.windows_server_instance_profile
```
Comment (uddr) [inline jenkins/windows-server/shared.pkr.hcl:47]: would be good to format this block

File: jenkins/windows-server/shared.pkr.hcl [hunk 2]
```diff
@@ -86,6 +88,7 @@ source "amazon-ebs" "windows-server-2022" {
...
```
Comment (uddr) [inline jenkins/windows-server/shared.pkr.hcl:91]: would be good to format this block

File: jenkins/windows-server/jenkins-agent.pkr.hcl [hunk 3]
```diff
@@ -1,24 +1,33 @@
...
     only  = ["amazon-ebs.jenkins-agent-windows-server-2022"]
     inline = ["& 'C:/Program Files/Amazon/EC2Launch/EC2Launch.exe' status --block"]
   }
   provisioner "powershell" {
     inline = ["C:/Windows/Temp/scripts/base.ps1"]
   }
+  provisioner "powershell" {
+    elevated_password = var.windows_server_winrm_password
+    elevated_user     = local.user_name
+    inline = ["C:/Windows/Temp/scripts/openssh.ps1 -configfiles C:\\config"]
```
Comment (uddr) [inline jenkins/windows-server/jenkins-agent.pkr.hcl]: would be good to format this block

**Review by uddr — APPROVED** [review]
lgtm

**Comment by flichtenheld** [comment]
Ran packer fmt on all files

**Pull request merged**